AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -159,6 +208,13 @@ extension CardViewController {
             additionalItem.identifier = ViewIdentifierBuilder.build(scopeInstance: scope, postfix: "additionalAuthPasswordItem")
             additionalItem.keyboardType = .numberPad
             additionalItem.isVisible = configuration.koreanAuthenticationMode == .show
+            
+            additionalItem.onDidBeginEditing = { [weak self] in
```

Comment:
A lot of repetitions, may I suggest extracting these two calls into a helper method? Something similar to what you've done in the tests.

`func setupFocusEvents(for control: UIControl, target: AnalyticsEventTarget)`